src/perfEvents_linux.cpp

```diff
@@ -557,7 +557,7 @@ int PerfEvents::createForThread(int tid) {
     if (FdTransferClient::hasPeer()) {
         fd = FdTransferClient::requestPerfFd(&tid, &attr);
     } else {
-        fd = syscall(__NR_perf_event_open, &attr, tid, -1, -1, 0);
+        fd = syscall(__NR_perf_event_open, &attr, tid, cpu, -1, 0);
```

Comment:
When setting `cpu` argument here, it also needs to be supported in FdTransfer protocol. Otherwise, there will be an inconsistent behavior between using and not using `--fdtransfer` option.